misc/python/materialize/benchmark_utils.py

```diff
@@ -13,7 +13,7 @@
 
 
 def resolve_tag_of_common_ancestor(tag_when_on_default_branch: str = "latest") -> str:
-    if buildkite.is_on_default_branch():
+    if buildkite.is_in_buildkite() and not buildkite.is_in_pull_request():
```

Comment:
This does more than extend docs

Reply:
Yes, you are right.

(Though, the change should be semantic-preserving™️.)